Sources/Tracking/CustomerIOImplementation+ScreenViews.swift

```diff
@@ -2,7 +2,9 @@ import Foundation
 #if canImport(UIKit)
 import UIKit
 
-@available(iOSApplicationExtension, unavailable) // screen view tracking is not available for notification service extension. disable all functions having to deal with screen view tracking feature. 
+@available(iOSApplicationExtension, unavailable)
+// screen view tracking is not available for notification service extension.
+// disable all functions having to deal with screen view tracking feature.
```

Comment:
```suggestion
// screen view tracking is not available for notification service extension.
// disable all functions having to deal with screen view tracking feature.
@available(iOSApplicationExtension, unavailable)
```

I can see this breaking this code - having the comments *below* the `@available` code.